src/WebJobs.Extensions.Http/HttpBindingApplicationBuilderExtension.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Http
+{
+    public static class HttpBindingApplicationBuilderExtension
+    {
+        public static IApplicationBuilder UseHttpBindingRouting(this IApplicationBuilder builder, IApplicationLifetime applicationLifetime, Action<WebJobsRouteBuilder> routes)
```

Comment:
I'd propose `UseHttpBinding` to make it clear this is for the binding/WebJobs (so it doesn't get missed in the context of ASP.NET)